internal/services/workloads/workloads_sap_virtual_instance_resource_test.go

```diff
@@ -0,0 +1,1122 @@
+		t.Skip("Skipping as `ARM_TEST_SAP_VIRTUAL_INSTANCE_NAME` and `ARM_TEST_CENTRAL_SERVER_VM_ID` are not specified")
+	}
+
+	data := acceptance.BuildTestData(t, "azurerm_workloads_sap_virtual_instance", "test")
+	r := WorkloadsSAPVirtualInstanceResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.discoveryConfiguration(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep(),
+	})
+}
+
+func (r WorkloadsSAPVirtualInstanceResource) Exists(ctx context.Context, clients *clients.Client, state *pluginsdk.InstanceState) (*bool, error) {
+	id, err := sapvirtualinstances.ParseSapVirtualInstanceID(state.ID)
+	if err != nil {
+		return nil, err
+	}
+
+	client := clients.Workloads.SAPVirtualInstances
+	resp, err := client.Get(ctx, *id)
+	if err != nil {
+		if response.WasNotFound(resp.HttpResponse) {
+			return utils.Bool(false), nil
+		}
+		return nil, fmt.Errorf("retrieving %s: %+v", id, err)
+	}
+	return utils.Bool(resp.Model != nil), nil
+}
+
+func (r WorkloadsSAPVirtualInstanceResource) template(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+provider "azurerm" {
+  features {
+    resource_group {
+      prevent_deletion_if_contains_resources = false
+    }
+  }
+}
+
+data "azurerm_subscription" "current" {}
+
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-sapvis-%d"
+  location = "%s"
+}
+
+resource "azurerm_user_assigned_identity" "test" {
+  name                = "acctest-uai-%d"
+  location            = azurerm_resource_group.test.location
+  resource_group_name = azurerm_resource_group.test.name
+}
+
+resource "azurerm_role_assignment" "test" {
+  scope                = data.azurerm_subscription.current.id
+  role_definition_name = "Azure Center for SAP solutions service role"
+  principal_id         = azurerm_user_assigned_identity.test.principal_id
+}
+
+resource "azurerm_virtual_network" "test" {
+  name                = "acctest-vnet-%d"
+  address_space       = ["10.0.0.0/16"]
+  location            = azurerm_resource_group.test.location
+  resource_group_name = azurerm_resource_group.test.name
+}
+
+resource "azurerm_subnet" "test" {
+  name                 = "acctest-subnet-%d"
+  resource_group_name  = azurerm_resource_group.test.name
+  virtual_network_name = azurerm_virtual_network.test.name
+  address_prefixes     = ["10.0.2.0/24"]
+}
+
+resource "azurerm_resource_group" "app" {
+  name     = "acctestRG-sapapp-%d"
+  location = "%s"
+
+  depends_on = [
+    azurerm_subnet.test
+  ]
+}
+`, data.RandomInteger, data.Locations.Primary, data.RandomInteger, data.RandomInteger, data.RandomInteger, data.RandomInteger, data.Locations.Primary)
+}
+
+func (r WorkloadsSAPVirtualInstanceResource) basic(data acceptance.TestData, sapVISNameSuffix int) string {
+	return fmt.Sprintf(`
+%s
+
+resource "azurerm_workloads_sap_virtual_instance" "test" {
+  name                        = "X%d"
```

Comment:
this should be `acctest` as in every other resource

Reply:
Service API indicates that it should start with "X".